manifests/params.pp

```diff
@@ -68,7 +68,7 @@
   }
 
   # Zabbix overall params. Is used by all components.
-  $zabbix_package_state                     = 'present'
+  $zabbix_package_state                     = 'latest'
```

Comment:
this is for the use case that somebody runs zabbix 3.0 or 2.4 and wants to update to 3.2.